frontend/components/ActivityDetails/InstallDetails/VppInstallDetailsModal/VppInstallDetailsModal.tsx

```
@@ -532,8 +531,7 @@ export const VppInstallDetailsModal = ({
         />
         {isVerificationTimedOut && (
           <p>
-            If the app is installed later, Fleet will update the status when the
-            host is refetched.
+            If the install finishes later, Fleet will update the status when the host is refetched.
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)